### PR TITLE
Remove jquery

### PR DESCRIPTION
**Pull request by miguel-guedes**

### CODE REVIEW

File: src/dom.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
+// @flow
+
+import { Css } from './consts';
+
+export type Position = {| x: number, y: number |};
+
+function classNameToSet(el: HTMLElement): Set<string> {
```
Comment (kujenga) [inline src/dom.js]: Is there a good way to add some unit test coverage for the functions in this file?

Comment (miguel-guedes) [reply src/dom.js]: Yes, it's simple enough: create `dom.test.js` and voila.  Should it be done in this PR or a new one though, @kujenga ?

Comment (kujenga) [reply src/dom.js]: A new PR sounds good to me 👍

Comment (miguel-guedes) [reply src/dom.js]: voila = add test cases

**Pull request merged**